scss/_nav.scss

```diff
@@ -124,24 +145,30 @@
 //
 
 .nav-pills {
```

Comment:
Pills looks weird in this PR

Reply:
Yep due to https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1181#discussion_r847164806